spec/support/shared_examples/message_resolver_failed.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.shared_examples 'message resolver failed' do
+  it 'sets error content' do
+    expect(message_resolver.content).to eq(expected_resolver_message)
+  end
+
+  it 'still sets the callback' do
+    expect(message_resolver.callback).to be_an_instance_of(DiscordEngine::InteractionCallback)
+  end
```

Comment:
we should probably also ensure that the callback is of the type we need, WDYT?

Reply:
nice one! Done!